srcpkgs/onedrive/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'onedrive'
+pkgname=onedrive
+version=2.4.3
+revision=1
+wrksrc="onedrive-${version}"
+build_style=gnu-configure
+configure_args="DC=ldc2 --bindir=/usr/bin"
+depends="curl"
+hostmakedepends="ldc pkg-config"
+makedepends="libcurl-devel sqlite-devel libnotify-devel"
+short_desc="Maintained client for Microsoft OneDrive"
+maintainer="selfisekai <laura@selfisekai.rocks>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/abraunegg/onedrive"
+changelog="https://raw.githubusercontent.com/abraunegg/onedrive/v${version}/CHANGELOG.md"
+distfiles="https://github.com/abraunegg/onedrive/archive/v${version}.tar.gz"
+checksum=4236f436b3658e360164d945a8bc5c9620539c1339ef9182e3d060d2e3b90b9b
```

Comment:
Add `nocross="uses host flags for cross compilations"` because this will not cross build.